pkg/driver/camera/camera_linux.go

```diff
@@ -158,8 +161,13 @@ func (c *camera) Open() error {
 		return err
 	}
 
-	// Late frames should be discarded. Buffering should be handled in higher level.
-	cam.SetBufferCount(1)
+	// Buffering should be handled in higher level.
```

Comment:
Removed "Late frames should be discarded." since we this PR discards late frames.